JuliaGPU/session2/Videh Raj Nema/q3.jl

```diff
@@ -0,0 +1,43 @@
+function squareroot(num)
+           start=0
+           stop=num
+           ans=0
+           while start <= stop
+               mid=div((start + stop),2)
+               if mid*mid==num
+                   ans=mid 
+                   break
+               end
+               if mid*mid<num 
+                   start = mid + 1 
+                   ans = mid
+               else
+                   stop = mid - 1 
+               end
+           end
+           
+           inc=0.1; 
+           for i=0:9 
+               while ans*ans<=num 
+                   ans+=inc
+               end
+               ans=ans-inc 
+               inc=inc/10     
+           end
+           return ans
+           end
+
+a=rand(2^20)
+b=rand(2^20)
+
+array_sqrt!(num)=return squareroot(num)      
+
+a=array_sqrt!.(a)
```

Comment:
You are almost right. The question asks you to define array_sqrt to take sqrt of an array, not it's broadcast version to do the same. Here array_sqrt is simply the same as squareroot. You need to make a small change